UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -620,7 +620,7 @@ private void ExternalMetaboliseReactions()
 				{
 					foreach (var surfaceReagent in SurfaceReagents)
 					{
-						ApplyReagentsToSurface(node.SmokeNode.Present.Clone(), surfaceReagent.Key);
+						ApplyReagentsToSurface(node.FoamNode.Present.Clone(), surfaceReagent.Key);
```

Comment:
why would mobs have foam on their surfaces? also what happens if we have different surface reagents that isn't foam?

Reply:
well if foam is capable of making walls I imagine it's like fills up the entire Tile and you have to walk through it

Reply:
surface reagents ? what do you mean like chemical patch?